GUIDELINES.md

```diff
@@ -0,0 +1,140 @@
+# Fulma : contributing to the project
+
+## Foreword
+Hi there, we're happy you deciced to join the band so as to make this project even better!
+Although contributing to this project is not complicated we'll help you get started.
+
+# Build requirements
+
+* [dotnet SDK](https://www.microsoft.com/net/download/core) 2.0 or higher
+* [node.js](https://nodejs.org) 6.11 or higher
+* Yarn JS package manager: [yarn](https://yarnpkg.com)
+
+## Install dependencies and first build
+
+In order to install the dependencies, generate the docs and build the project just type the following commands:
+
+On windows:
+```shell
+./build.cmd
+```
+
+On Unix:
```

Comment:
Just a comment regarding this change: 
I would not say MacOS-Whatever is Unix nor would I ever tell anybody that Linux is Unix.
I think people may get hanged in some countries just for telling such crazy things 😉 
But hey, I understand it's funny to summon some ancient trolls sometimes :trollface:

Reply:
Stop playing with the name, I already have Thot name problem to fix somehow...